kotlin-reference-server/src/main/kotlin/org/stellar/reference/callbacks/customer/CustomerService.kt

```diff
@@ -55,11 +55,18 @@ class CustomerService(private val customerRepository: CustomerRepository) {
         customer.copy(
```

Comment:
💡 Is there anything like [BeanUtils ](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/beans/BeanUtils.html#copyProperties(java.lang.Object,java.lang.Object,java.lang.Class))in Kotlin?

Reply:
Or use `gson.toJson` and `gson.fromJson` for copying? 

I think this is for the reference and does not require critical performance but rather explanations.

Reply:
I wrote it this way since I don't want `null` request fields to delete a previously set customer field. By default, `BeanUtils#copyProperties` does not ignore `null` fields, we would need to write a wrapper around it if we want this behavior. This is the same reason, I did not use `gson` here.